source/inputCore.py

```diff
@@ -452,6 +452,17 @@ def __eq__(self, other: Any) -> bool:
 		return NotImplemented
 
 
+decide_handleRawKey = extensionPoints.Decider()
```

Comment:
could unit tests be added for this?

Reply:
I thought about this, but the actual functionality lives in the keyboard handler, it's just exposed in the input core for consistency. The keyboard handler doesn't currently seem to have any test suite I can find though (I could totally be overlooking something obvious :) )

We can and should add tests for the keyboard handler if there aren't any, but I don't necessarily think they should go in this PR.

Reply:
Thanks, I agree